wiki/Ranking_Criteria/skinning/en.md

```diff
@@ -0,0 +1,1002 @@
+Skin Set List 
+============= 
+
+The following table contains all skin sets that are presented in the draft worked out by the skinning subdivision of the [UBKRC-Project](https://osu.ppy.sh/forum/t/480331) (name credits go to Ephemeral)
+
+<table cellpadding="0" cellspacing="0">
```

Comment:
I can, but that would blow the entire listing out of proportion by a big shot and decrease readability and usability by adding a lot of redundant information, but it's possible